src/libs/Network.js

```diff
@@ -3,9 +3,13 @@ import lodashGet from 'lodash/get';
 import Onyx from 'react-native-onyx';
 import HttpUtils from './HttpUtils';
 import ONYXKEYS from '../ONYXKEYS';
+import * as ActiveClientManager from './ActiveClientManager';
 
 let isQueuePaused = false;
 
+// Are we retrying requests when user comes back online?
+let retryRequestsAdded = false;
```

Comment:
The comment above doesn't really help explain what the code is doing or how it works.